### PR TITLE
Sorry! I should have tested before requesting... Here's the right CDN link for 1.3.6

### PR DESCRIPTION
**Pull request by Tysseract**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,14 +8,14 @@ RUN addgroup minecraft && \
...
+RUN wget "https://media.forgecdn.net/files/2981/831/MCEternal(ServerPack1.3.6).zip" -O server.zip && \
         unzip server.zip -d server_zip && \
         mv -v ./server_zip/*/* ./ && rm -rf server.zip server_zip && \
         chown -R minecraft:minecraft /srv/minecraft
 COPY --chown=minecraft:minecraft ./ServerStart.sh /
 
 USER minecraft
 
-EXPOSE 25565
+EXPOSE 25565 25575 8314
```
Comment (mymindstorm) [inline Dockerfile]: Why are you exposing more ports?

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Run the container and publish port 25565. You can mount `/srv/minecraft` to a vo
 $ docker volume create mc-eternal-vol
 mc-eternal-vol
 # Create container and mount Minecraft data to volume
-$ docker run -v mc-eternal-vol:/srv/minecraft -p 25565:25565 -i -d mymindstorm/mc-eternal:1.3.3
+$ docker run -v mc-eternal-vol:/srv/minecraft -p 25566:25565 -p 25576:25575 -p 8124:8123 -i -d --name mceternal tysseract/mc-eternal:1.3.6
```
Comment (mymindstorm) [inline README.md]: `tysseract/mc-eternal` is not the name of the docker container that will be published. Please revert this.

File: Dockerfile [hunk 3]
```diff
@@ -8,14 +8,14 @@ RUN addgroup minecraft && \
 
 WORKDIR /srv/minecraft
 
-RUN wget "https://www.curseforge.com/minecraft/modpacks/minecraft-eternal/download/2981831/file" -O server.zip && \
+RUN wget "https://media.forgecdn.net/files/2981/831/MCEternal(ServerPack1.3.6).zip" -O server.zip && \
```
Comment (mymindstorm) [inline Dockerfile]:
```suggestion
RUN wget "https://media.forgecdn.net/files/2981/831/MCEternal(ServerPack${VERSION}).zip" -O server.zip && \
```

**Review by mymindstorm — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Tysseract) [reply README.md]: I'm not very good at GitHub I didn't mean for this to get committed...

File: Dockerfile (hunk 1 above)
Comment (Tysseract) [reply Dockerfile]: I actually meant to pull request the previous commit but 25575 is for RCON and 8123 is for dynmap

**Comment by Tysseract** [comment]
I didn't realize commits made after the pull request added to the pull request but it should be better now. Also, port 25575 is for RCON (native to Minecraft remote control and disabled in settings by default) and 8123 is for dynmap (a web-based map mod)...

**Comment by mymindstorm** [comment]
Please remove the extra ports. Only 25565 should be exposed. rcon and dynmap are not part of the default Minecraft server config and should not be exposed be default. If they are enabled, then users can expose the ports themselves using -p.

**Pull request closed without merging**